lib/fizzbuzz.rb

```diff
@@ -1,12 +1,20 @@
 class SuperFizzBuzz
-
-  def run(input)
-
-    #Implement your code here
+  def run(x)
+    output = ""
+    if x % 3 == 0
+      output += "Fizz"
+    end
+    if x % 5 == 0
+      output += "Buzz"
+    end
+    if output == ""
+      output = x
+    end
+    return output
```

Comment:
I like the approach here.  Some one-liner if-statements (such as `output += 'Fizz' if x % 3 == 0`) would make it really slick.